kwanghwi/myapp/app.js

```diff
@@ -0,0 +1,63 @@
+const http = require("http");
+const express = require("express");
+const cors = require("cors");
+const morgan = require("morgan");
+
+const dotenv = require("dotenv");
+dotenv.config();
+
+const { DataSource } = require("typeorm");
+
+const appDataSource = new DataSource({
+  type: process.env.TYPEORM_CONNECTION,
+  host: process.env.TYPEORM_HOST,
+  port: process.env.TYPEORM_PORT,
+  username: process.env.TYPEORM_USERNAME,
+  password: process.env.TYPEORM_PASSWORD,
+  database: process.env.TYPEORM_DATABASE,
+});
+
+appDataSource
+  .initialize()
+  .then(() => {
+    console.log("Data Source has been initialized!");
+  })
+  .catch(() => {
+    console.log("Promise Rejected!");
+  });
+
+const app = express();
+const PORT = process.env.PORT;
+
+app.use(express.json());
+app.use(cors());
+app.use(morgan("dev"));
+
+app.get("/ping", (req, res) => {
+  return res.status(200).json({ message: "pong" });
+});
+
+app.post("/user/signup", async (req, res) => {
+  const { userId, password, email, profile_image } = req.body;
```

Comment:
자바스크립트에서는 camelCase를 사용하여 단어간 띄어쓰기를 구분하고 있습니다.

지금 41번 코드를 보면 camelCase와 snake_case가 혼용되고 있습니다.

다른 부분도 camelCase로 통일하여 주면 좋을 것 같네요!

```suggestion
  const { userId, password, email, profileImage } = req.body;
```